playwright/pages/storybook.ts

```diff
@@ -38,11 +38,11 @@ export class StorybookStoryPage {
     }
 
     async expectFullPageScreenshot(): Promise<void> {
-        await expect(this.page).toHaveScreenshot({ maxDiffPixelRatio: 0.01 })
+        await expect(this.page).toHaveScreenshot({ maxDiffPixelRatio: 0.03 })
```

Comment:
Unfortunately this wouldn't affect our current issues, because the methods changed in this PR are not used in the problematic `insights.spec.ts`